src/BuiltinExtensions/ComfyUIBackend/WorkflowGenerator.cs

```diff
@@ -371,16 +378,20 @@ public JArray RecompositeCropped(string boundsNode, JArray croppedMask, JArray f
             ["upscale_method"] = "bilinear",
             ["crop"] = "disabled"
         });
-        string thresholded = CreateNode("ThresholdMask", new JObject()
```

Comment:
Feathering doesn't sound bad, but don't get rid of the threshold, that's there very intentionally - removing it will cause obscure issue regressions.

The feathering itself should probably be a separate addition from the rest of this PR being a bugfix, and be controlled by a parameter